mappings/net/minecraft/world/biome/Biome.mapping

```diff
@@ -1,7 +1,11 @@
 CLASS net/minecraft/class_1959 net/minecraft/world/biome/Biome
 	FIELD field_20335 temperatureCache Ljava/lang/ThreadLocal;
 	FIELD field_22039 effects Lnet/minecraft/class_4763;
+	FIELD field_24677 REGISTRY_CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_25819 CODEC Lcom/mojang/serialization/MapCodec;
```

Comment:
hmm, since this is a map codec, is there cases where like there is a json object which you can derive both a biome and some other stuff from that one json object? I assume this exists so the registry codec can either dispatch to an id field lookup or pass the object to this map codec?

Reply:
I don't understand much about codecs, but the registry codec here is a registry element codec cast to a codec, hence the name